### PR TITLE
Select layer on which instances will be added by default

### PR DESCRIPTION
**Pull request by AlexandreSi**
As discussed with @LuniMoon, it's not the prettiest, but it might be revamped in the future.

image

### CODE REVIEW

File: newIDE/app/src/SceneEditor/index.js [hunk 1]
```diff
@@ -491,22 +493,6 @@ export default class SceneEditor extends React.Component<Props, State> {
     else this.toggleObjectsList();
   };
 
-  _onAddInstanceUnderCursor = () => {
```
Comment (ClementPasteau) [inline newIDE/app/src/SceneEditor/index.js:494]: uh, unused?

**Review by ClementPasteau — APPROVED** [review]
🚀

File: newIDE/app/src/SceneEditor/index.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/SceneEditor/index.js:494]: Good catch indeed

Comment (AlexandreSi) [reply newIDE/app/src/SceneEditor/index.js:494]: Yep, I don't know when it was abandoned

**Review by 4ian — APPROVED** [review]
Looks great to me!

**Pull request merged**